SVA Batch Output

anikde07 opened the pull request:
Addresses issue #220 
Extracted output and warnings/messages. Additional formatting added for the rendering box element

anikde07 committed:
Created bounding boxes for ouput and warnings/messages. Added console messages/warnings and output

LeaSeep reviewed (changes requested):
That is a very nice start.
Would have some suggestions to overall improve a bit clarity as now the unfamiliar user is faced with a lot of text:
- can we might introduce like a toggle options to get the advanced output and warning or messages?
- somehow highlight that those information come with the batch correction
- add some help text for interpretation: such as: "Here you can see what the sva-based batch correction was doing ..."
- Could you extract the number of covariates/batches found and put that to the top-level information ? E.g. under " The processed data has dimensions: ..."
- text of the output can overflow the text box - there was some solution to this I think using a different shiny output object

anikde07 committed:
Added toggle for additional messages/warnings and added number of batch numbers to statistics panel

anikde07 commented:
I fixed the design of the toggle and moved the number of batches found to the statistics box.
Could you review this and let me know if this is working for you?

I am trying to figure out still how to not show the toggle when no batch correction has been performed

anikde07 committed:
Fixed issue of toggle showing up when no batch correction is selected

anikde07 commented:
Changed the default behavior of the toggle. Can you check if this works? I will then merge the PR

anikde07 committed:
Changed some comments

LeaSeep reviewed (approved):
tested it - looks all good!
